demo/danesfield_demo.py

```diff
@@ -0,0 +1,134 @@
+import os
+import sys
+import subprocess
+import boto3
+from collections import defaultdict
+import argparse
+
+def run(args):
+	parser = argparse.ArgumentParser(description="Download WV3 images to run the Danesfield pipeline")
+	parser.add_argument('--site', help='CORE3D site to download images for', \
+						default='Jacksonville')
+	parser.add_argument('--img_path', help='Location where WV3 images should be downloaded to', \
+						default=f'{os.getcwd()}/imgpath')
+	parser.add_argument('--run', help='Run Danesfield after downloading images', \
+						action='store_true')
+	parser.add_argument('--out_path', help='Location that Danesfield should use as an output directory', default='outdir')
+	parser.add_argument('--config_dir', help='Location where Danesfield configuration files can be found. \
+											You must use input_<region>.ini and <region>_config.json for \
+											Danesfield and Vissat config names, respectively.', default='configdir')
+	parser.add_argument('--show_sites', help='List available CORE3D sites and exit', \
+						action='store_true')
+	parser.add_argument('--model_path', help='Location of necessary model files for Danesfield', default="Models")
```

Comment:
Consider communicating default values to the user, e.g. in help='a short description of the parameter; default: %(default)s'